src/Tribe/RSVP/Early_Access/Update_Notice.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Tribe\Tickets\RSVP\Early_Access;
+
+use Tribe__Tickets__Main;
+use Tribe__Settings;
+use Tribe__Admin__Notices;
+
+class Update_Notice {
+
```

Comment:
:no_entry_sign: Missing doc comment for class Update_Notice

<sub>hash: be029522be24735fb56fce87ce4be201</sub>